### DISCUSSION TITLE
[Additional prop] Add "isIncluded" as a prop for Show & Hide components

### DISCUSSION THREAD

**Discussion by VaZark** [chakra-ui/chakra-ui#7634] [Q&A]
The show and Hide components are extremely useful when dealing with responsive layouts.

```jsx
// Example from our codebase
<Show above="md"> desktop Children </Show>
<Show below="md"> Tablet & Mobile Children </Show>
```

However, by definition, the `above` and `below` props both include the breakpoint when rendering their children.

Thus, at exactly the breakpoint (here at _md_), all children are visible. It'd be great if we could just add an `isIncluded ` to be used that subtracts 1px from the breakpoint when _false_

**Comment by stevenqzhang** [comment]
I have this exact problem as well. @VaZark have you found any good workarounds?

I think the fix at a high level fairly simple

1. Add isIncluded here https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/media-query/src/show.tsx#L29

2. Plumb 'isIncluded' through `getBreakpoint` https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/media-query/src/media-query.tsx#L13
